### ISSUE TITLE
Missing HTTP headers using file provider and Traefik2.0-alpha

### ISSUE THREAD

**Issue by dniel** [containous/traefik#4659]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
I have tried to configure the file provider for Traefik2.0-alpha and when using the whoami container to debug I see that some HTTP headers that Traefik 1.7 added (X-forwarded-host, x-forwarded-method, x-forwarded-proto, x-forwarded-uri.) is not there. The only header still present is X-Forwarded-for.

### What did you expect to see?
HTTP headers x-forwarded-for, x-forwarded-host, x-forwarded-method, x-forwarded-proto, and x-forwarded-uri.

### What did you see instead?
Only the header x-forwarded-for

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      2.0.0-alpha2
Codename:     faisselle
Go version:   go1.12.1
Built:        2019-03-19T18:44:59Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
################################################################
# File Provider
################################################################

[http]
  ################################################################
  # Middlewares
  ################################################################
  [http.middlewares]
      [http.middlewares.forwardauth.ForwardAuth]
        address = "http://host.docker.internal:8080/authorize"
        authResponseHeaders =  ["Authorization", "X-FORWARDAUTH-NAME", "X-FORWARDAUTH-SUB", "X-FORWARDAUTH-EMAIL"]


  ################################################################
  # Routers
  ################################################################
  [http.routers]
    [http.routers.router0]
      entrypoints = ["http"]
      service = "whoami"
      rule = "Host(`whoami.example.test`)"

    [http.routers.router1]
      entryPoints = ["http"]
      priority = 9999999
      service = "oauth"
      rule = "Host(`auth.example.test`)"


  ################################################################
  # Services
  ################################################################
  [http.services]
    [http.services.whoami]
      [http.services.whoami.LoadBalancer]
        [[http.services.whoami.LoadBalancer.Servers]]
          url = "http://host.docker.internal:8000"
          weight = 1

    [http.services.oauth]
      [http.services.oauth.LoadBalancer]
        [[http.services.oauth.LoadBalancer.Servers]]
          url = "http://host.docker.internal:8080"
          weight = 1
```

Output from whoami
```
Hostname: 9c754487d738
IP: 127.0.0.1
IP: 172.17.0.3
GET / HTTP/1.1
Host: host.docker.internal:8000
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.5
Cache-Control: no-cache
Cookie: AUTH_NONCE=4d060c125a784ac4a24044392749fb42
Pragma: no-cache
Upgrade-Insecure-Requests: 1
X-Forwarded-For: 172.17.0.1
```

**Comment by dniel** [comment]
would it be possible to add a new X-Forwarded-Route field? with the name of the route that received the request?

**Comment by mpl** [comment]
@dniel that is a different point that might need careful thinking. Could you please open another issue about it?

**Comment by dniel** [comment]
Did that for Traefik1 in september last year #3919. Thought this could be simpler and maybe even a good time to add it when you already where working with something similar.

**Comment by ldez** [comment]
A tracing tool (Jeager, Zipkin, Datadog, ...) seems a better solution than add a non standard header.

With the v2 (and the v1) you can use the header middleware to add a header with name of the route.

https://docs.traefik.io/v2.0/middlewares/headers/

**Comment by dniel** [comment]
@ldez its not for tracing,  have a look at #3919 to see what I would use such field for.

**Comment by traefiker** [comment]
Closed by #4707.
